src/setuptools_scm/hg.py

```diff
@@ -68,15 +70,23 @@ def parse(root, config=None):
         pass  # unpacking failed, old hg
 
 
-def get_latest_normalizable_tag(root):
-    # Gets all tags containing a '.' (see #229) from oldest to newest
+def get_latest_normalizable_tag(config):
```

Comment:
I don't think that changing the signature of a public function is a good idea.

Reply:
good catch